### PR TITLE
Docs 4952 override secure property

### PR DESCRIPTION
**Pull request by valkyrie69**

### CODE REVIEW

**Review by kunli-mulesoft — APPROVED** [review]
@valkyrie69 , this change looks good to me. Thanks a lot!

File: modules/ROOT/pages/secure-application-properties.adoc [hunk 1]
```diff
@@ -192,26 +195,32 @@ Then, at deployment time, the operator can type in the `secure.key` value into t
...
+Encrypted properties that are bundled with a Mule application inside a secure properties placeholder file do not appear in the Runtime Manager *Properties* tab. This keeps the values safely locked inside the Mule application.
+
+The Runtime Manager console does not have access to the secret key, so if you need to update an encrypted value, you must open the Mule application source files, recode the new encrypted value with the secret key, then regenerate the Mule application's deployable archive. 
+
+In many environments, it is not possible for you to do this yourself, which means you have send the Mule application back to the developer to do it for you. However, there is a way to design a Mule application to allow encrypted properties to be overridden safely.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/secure-application-properties.adoc]: "have to send"

File: modules/ROOT/pages/secure-application-properties.adoc [hunk 2]
```diff
@@ -192,26 +195,32 @@ Then, at deployment time, the operator can type in the `secure.key` value into t
...
+So for any encrypted property that is also marked as hidden in the Mule application's `secureProperties` entry, you can safely replace any encrypted property with a clear-text value. The clear-text value is then securely stored in the CloudHub properties database, and securely passed into the Mule application every time the Mule application is started.
 
-The new value is also hidden once the application is deployed and can never be viewed again.
+The new value is hidden once the application is deployed and can never be viewed again.
 
-This means that when you override a secure property in the CloudHub properties tab for the application, the values never needs to be encrypted. In this scenario, securing the values of sensitive properties is reduced to controlling which operators have access to those values when they deploy or redeploy the secure application.
+This means that when you override a secure property in the CloudHub *Properties* tab for the application, the value never needs to be encrypted. In this scenario, only users who have access to the values when they deploy or redeploy a secure application can override secure properties.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/secure-application-properties.adoc]: This what? Do we mean Thus or Therefore? Also (okay to punt), I'm not really sure what we're saying with the second sentence. What does it mean to have access to the values? Are we saying anyone with permission to edit can override any secure property (but they can't view it once deployed)?

**Review by Mulesoft-Mysti — APPROVED** [review]
1 missing word, 2 questions.

**Pull request merged**